quest/util/misc.py

```diff
@@ -88,17 +100,35 @@ def bbox2poly(x1, y1, x2, y2, reverse_order=False, as_geojson=False, as_shapely=
 
 
 def build_smtk(smtk_subdir, smtk_filename, **kwargs):
-    """build smtk file from template and kwargs and stream as string.
+    """Build smtk file from the template, kwargs, and stream as a string.
+
+    Args:
+        smtk_subdir (string):
```

Comment:
smtk_subdir (string): subdirectory where SMTK template is stored.
smtk_filename (string): name of the SMTK template file.
 kwargs (kwargs): Keyword arguments to render the SMTK template with.